content/phishing.md

```diff
@@ -0,0 +1,178 @@
++++
+title = "Phishing / falske e-poster"
+lastmod = 2020-08-28T12:00:00
+[menu]
+main = { parent = "Teknologi" }
++++
+
+{{% ingress %}}
+
+Av og til mottar vi falske e-poster som har til hensikt å enten lure til seg informasjon, svindle mottakeren for en betaling eller skade datamaskinen. Dette kalles phishing. I denne artikkelen kan du lære om hvordan du kan identifisere en slik e-post.
```

Comment:
Denne filen trenger deling av lange linjer for å bestå Markdownlint-sjekken. Jeg vil tro at editoren din har en innebygget kommando for å brette linjer innad i et avsnitt, så du slipper å gjøre det manuelt.